src/arviz_plots/plots/distplot.py

```diff
@@ -352,7 +374,7 @@ def plot_dist(
             labeller=labeller,
             **title_kwargs,
         )
-    if (kind == "kde") and (plot_kwargs.get("remove_axis", True) is not False):
+    if (kind in ("kde", "hist")) and (plot_kwargs.get("remove_axis", True) is not False):
```

Comment:
I would modify this logic. Instead, I would set the value of the `remove_axis` depending on `kind` at the top, right after copying `plot_kwargs`.

```python
if kind in ("hist", "ecdf"):
    plot_kwargs.setdefault("remove_axis", False)
```

I think as a default we'll want to remove the axis for kde and quantile dotplot but keep it for the others.